src/AnalyticsEngine/App.ControlPanel.Engine/SqlExtentions/Profiling-03-CreateSchema.sql

```diff
@@ -1187,7 +1306,17 @@ BEGIN
         [Yammer Used WinPhone] [bit] NOT NULL DEFAULT 0,
         [Yammer Used Android] [bit] NOT NULL DEFAULT 0,
         [Yammer Used iPad] [bit] NOT NULL DEFAULT 0,
-        [Yammer Used iPhone] [bit] NOT NULL DEFAULT 0
+        [Yammer Used iPhone] [bit] NOT NULL DEFAULT 0,
+
+        -- Add COPILOT fields here
+
+        -- TODO
+        [Copilot ChatsCount] [bit] NOT NULL DEFAULT 0, 
+        [Copilot MeetingsCount] [bit] NOT NULL DEFAULT 0,
+        [Copilot FilesCount] [bit] NOT NULL DEFAULT 0
+
```

Comment:
Please use separate words for the metric names. Makes them friendlier and they don't need to be changed later in the report.